packages/vite-plugin-kit-routes/src/lib/plugin.ts

```diff
@@ -609,6 +642,18 @@ export function buildMetadata(
   return baseToReturn
 }
 
+function getSpValue(rawValue: string, param: ExplicitSearchParam) {
+  if ((param.arrayMode ?? 'join') === 'join') {
```

Comment:
I said before that default default should probably be `join`, but the more I think about it, the more I think that `split` should be default default.

`split` would be ok for you?

(I think I wrote this comment before, but can't find it :o)

Reply:
Well, I don't really care for the default. But I like that the generated code probably will be marginally more performant as the `String(...)` calls are only needed for params with the `join` mode and not for all params.

Will change it.